marklogic-data-hub-central/ui/e2e/cypress/integration/authorization.spec.tsx

```diff
@@ -42,7 +42,8 @@ describe('login', () => {
     cy.loginAsDeveloper()
     .wait(500)
     .url()
-    .should('include', '/home');
+     .should('include', '/tile')
+    // .should('include', '/home');
```

Comment:
Is there any benefit in keeping the commented-out code? I think it's almost always best to delete commented-out code - in the rare case something might be needed later, it can always be retrieved from version control.